micall/core/plot_contigs.py

```diff
@@ -500,9 +505,9 @@ def build_contig(reader,
             # Use up any events that went past the end of the contig.
             event_pos = event_positions.pop()
             for blast_num in blast_starts[event_pos]:
-                blast_ranges[blast_num - 1][0] = pos
+                blast_ranges[blast_num - 1][0] = end
```

Comment:
Is it possible to get coverage on this line, or is it an impossible scenario?

Reply:
I'm not sure how this can happen in real life, but theoretically this code catches a case where a blast match starts past the end of the contig. I've added a unit test for that case.